### PR TITLE
new Italian layout and some modifications to italian qwerty

### PR DESCRIPTION
**Pull request by nicoursi**
It adds a new italian layout with a number row included so that there is more space for more symbols.

### CODE REVIEW

File: addons/languages/italian/pack/src/main/res/xml/italian_qwerty_with_symbols.xml [hunk 1]
```diff
@@ -7,40 +7,40 @@
 
     <Row>
         <Key android:codes="q" android:popupCharacters="1¹₁" ask:hintLabel="1" android:keyEdgeFlags="left"/>
-        <Key android:codes="w" android:popupCharacters="2²₂ŵ" ask:hintLabel="2"/>
-        <Key android:codes="e" android:popupCharacters="3€³₃êèéëęē" ask:hintLabel="3"/>
```
Comment (menny) [inline addons/languages/italian/pack/src/main/res/xml/italian_qwerty_with_symbols.xml:11]: why did you remove so many popup characters? Usually, the `XXX_with_symbols` layouts have a lot of symbols all around.

Comment (nicoursi) [reply addons/languages/italian/pack/src/main/res/xml/italian_qwerty_with_symbols.xml:11]: I thought that I removed that commit but apparently I squashed it. I was supposed to push it after arena commit about reordering popup characters. Those diacritcs don't belong to Italian language, so they need to be removed if arena's commit gets merged.

Comment (menny) [reply addons/languages/italian/pack/src/main/res/xml/italian_qwerty_with_symbols.xml:11]: okay. for now, can you get them back?

**Comment by menny** [comment]
Why did you close this?

**Comment by nicoursi** [comment]
Sorry for closing everything, but as I didn't have time for this  anymore at the time, I preferred to close all the PRs and to archive my ASK fork, so not to receive failing tests notifications every day, many times a day.

**Comment by nicoursi** [comment]
There is also the English version of this new layout with a numbers row. The PR could not pass unity tests. I've seen a few English layouts PRs getting stale because of this reason. It would really be appreciated some feedback from the maintainers to get it passed.

**Review by menny — APPROVED** [review]
